src/app/services/identity.service.ts

```diff
@@ -15,7 +15,7 @@ export class IdentityService extends IonicIdentityVaultUser<DefaultSession> {
 
   constructor(public platform: Platform, private browserAuthPlugin: BrowserAuthPlugin) {
     super(platform, {
-      authMode: AuthMode.BiometricOnly,
+      // authMode: AuthMode.BiometricOnly,
```

Comment:
Tested this on iOS before - it worked with my Face Id. Reason to take it out? on Android, it simply didn't do anything.

Reply:
See https://github.com/ionic-team/ionifits/pull/22/commits/7faa13ad0a50431152afa682293fab3c5cb4f942